### ISSUE TITLE
Mobile Classifier Reskin

### ISSUE THREAD

**Issue by seanmiller26** [zooniverse/mobile#535]
A major step in the app redesign is to reskin the current classifier in order modernize it and make it feel like a more integrated part of the experience.

There are three main types of projects featured on mobile: a simple yes or no question with swipe, multiple choice question, and drawing tasks.

<img width="900" alt="Screen Shot 2024-03-06 at 2 16 31 PM" src="https://github.com/zooniverse/mobile/assets/89043489/17e8fe0c-0347-47ba-8685-5e3ad9e54655">

Mocked-up with all mobile elements:
<img width="250" alt="Screen Shot 2024-03-06 at 2 41 28 PM" src="https://github.com/zooniverse/mobile/assets/89043489/d3d50027-c020-47c1-b4a3-2d8f5c2e6dc5">

Tablet, portrait:
<img width="500" alt="Screen Shot 2024-03-06 at 2 43 43 PM" src="https://github.com/zooniverse/mobile/assets/89043489/764d0071-1139-4953-9cbb-36f347566b94">

Tablet, landscape:
<img width="700" alt="Screen Shot 2024-03-06 at 2 53 11 PM" src="https://github.com/zooniverse/mobile/assets/89043489/6ef522ed-187a-4886-bc09-b55270a72b61">

I am happy to open a dedicated issue for each task if that will help with specific details, otherwise everything can be found in the Figma file [here](https://www.figma.com/file/QC931orjhCdGoQogVg7iDZ/Mobile-App?type=design&node-id=0%3A1&mode=design&t=1XvNuTYDV6gq9PyN-1). 

As for workflow, these can be tackled one at a time. I have tried to reuse elements as much as possible, but obviously there are unique components for each task.

**Comment by coryzoon** [comment]
@lcjohnso @seanmiller26 Should there be any special considerations for "museum mode"? I think we currently apply a specific theme for projects that have museum mode turned on.

**Comment by lcjohnso** [comment]
> @lcjohnso @seanmiller26 Should there be any special considerations for "museum mode"? I think we currently apply a specific theme for projects that have museum mode turned on.

I don't believe so -- the look should be the same. The key difference when museum mode is enabled: that navigation should be locked -- the user shouldn't be able to tap on the Zooniverse Logo in the header and return to the homepage, nor open the sandwich menu to navigate to a different page.

**Comment by seanmiller26** [comment]
That's correct, all the styling will be similar in museum mode. 

@coryzoon you reminded me to open a related issue about disabling the home button, though.

**Comment by lcjohnso** [comment]
Closed by #568
